src/main.rs

```diff
@@ -97,12 +97,13 @@ impl RouteHandler for RoomHandler {
                 pg_conn.query_rows(
                     r#"
                     SELECT event_id, events.type, state_key, depth, sender, state_group,
-                        content, origin_server_ts,
+                        json, origin_server_ts,
                         array(
                             SELECT prev_event_id FROM event_edges
                             WHERE is_state = false and event_id = events.event_id
                         )
                     FROM events
+                    JOIN event_json USING (event_id)
```

Comment:
Can you put `LEFT JOIN` or `INNER JOIN` as otherwise I completely forget what the default is

Reply:
whereas I always have to remember that `INNER JOIN` means "normal" `JOIN`. but sure.